lib/paysafe/account_management_service.rb

```diff
@@ -56,6 +56,17 @@ def createMerchantAccount merchantAccount
       AccountManagement::MerchantAccount::new response
     end
 
+    def getMerchantAccountInfo merchantAccount
+      request = Request.new(
+        method: Request::GET,
+        uri: prepare_uri("/accounts/#{merchantAccount.merchantId}"),
+        query: { operationMode: 'consolidated' }
```

Comment:
I would rather pass mode as a parameter.

Reply:
it seems that query: { }  for parameters

Reply:
I’m talking about getMerchantAccountInfo method

Reply:
Or we can get it only with “consolidated” option?